GUI/MainWindow.xaml

```diff
@@ -391,6 +392,11 @@
                         
                         in .xml or .fasta format. Reference proteomes in .xml format can contain post-translational modification (PTM) locations,
                         whereas .fasta format only contains amino acid sequences.
+                       <Hyperlink NavigateUri="https://chemdata.nist.gov/dokuwiki/doku.php?id=peptidew:start" RequestNavigate="Hyperlink_RequestNavigate">
+                            Spectral libraries
+                       </Hyperlink>
+
+                       in .msp format can be added to improve search results.>
```

Comment:
There's an extra '>' on line 399

Reply:
Talking with Shortreed, he suggested a line break before the sentence on spectral libraries. I think this would be a good idea. Note that this will may required increasing the row height on line 377